lib/gollum.rb

```diff
@@ -17,7 +17,7 @@ module Gollum
   KEYBINDINGS = ['default', 'vim', 'emacs']
   
   ::I18n.available_locales = [:en]
-  ::I18n.load_path = Dir[::File.expand_path("lib/gollum/locales") + "/*.yml"]
+  ::I18n.load_path = Dir[::File.expand_path("../gollum/locales",  __FILE__) + "/*.yml"]
```

Comment:
🥲 I'm sorry.

Reply:
No worries! Also slipped through my review :) And once I had entangled this from the other issues, it was actually quickly discovered.